### PR TITLE
Fix print pass that contain more than one values

### LINKED ISSUE #1112: `print *, "a", x` fails
Here is a full example thanks to @Pranavchiku:
```fortran
program main

integer :: x(3)
x(1) = 1    
x(2) = 2
x(3) = 3
print *, "x :", x

end program
```
this prints:
```console
$ lfortran b.f90
x : 1841723168
```
While instead it should print:
```console
$ gfortran b.f90 && ./a.out
 x :           1           2           3
```

### PR DESCRIPTION
**Pull request by Smit-create**
Fixes #1112

### CODE REVIEW

**Comment by Smit-create** [comment]
This PR depends on #1132

**Review by certik — APPROVED** [review]
I think this looks good, thanks!

**Comment by certik** [comment]
Tests need to pass.

File: tests/reference/llvm-case_01-09dad75.stdout [hunk 1]
```diff
@@ -30,84 +30,12 @@ define i32 @main() {
...
   %i = alloca i32, align 4
   store i32 4, i32* %i, align 4
-  %0 = load i32, i32* %i, align 4
-  %1 = icmp eq i32 %0, 1
-  br i1 %1, label %then, label %else
-
-then:                                             ; preds = %.entry
   call void (i8*, ...) @_lfortran_printf(i8* getelementptr inbounds ([5 x i8], [5 x i8]* @3, i32 0, i32 0), i8* getelementptr inbounds ([2 x i8], [2 x i8]* @2, i32 0, i32 0), i8* getelementptr inbounds ([2 x i8], [2 x i8]* @1, i32 0, i32 0))
-  br label %ifcont9
-
```
Comment (Smit-create) [inline tests/reference/llvm-case_01-09dad75.stdout:40]: This deletion is actually a bug and is fixed in #1139

**Review by certik — APPROVED** [review]

**Pull request merged**